spec/transaction/paths.yaml

```diff
@@ -562,6 +714,7 @@ paths:
     put:
       tags:
         - Asset Upload Management
+        - Collection Asset Upload Management
```

Comment:
For my understanding: Is this tagged with "Collection Asset Upload Management" because you can also directly upload a part to a collection now, instead of only to an item?

Reply:
Still open, would you mind clarifying please?

Reply:
Yes. This path is where the parts are uploaded to s3. I decided to add the tag so it shows up in both tag sections.
Depending on the use-case you can either upload a part of an asset, or of a collection asset.